src/lib/utils/skin.ts

```diff
@@ -110,3 +111,19 @@ export function isSkin(asset: rgAsset): boolean {
               asset.market_hash_name.includes(keyword)
           );
 }
+
+export function getFadePercentage(asset: rgAsset, itemInfo: ItemInfo): number | undefined {
+    const FADE_TYPE_TO_CALCULATOR = {
+        Fade: FadeCalculator,
```

Comment:
Let's also quote `Fade` for consistency.

Reply:
i think the auto formatter doesn’t like it